### PR TITLE
zh-translation: docs/root/faq/debugging/why_is_envoy_sending_413s.rst

### PR DESCRIPTION
**Pull request by vgbhfive**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: docs/root/faq/debugging/why_is_envoy_sending_413s.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #699
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 .. _faq_why_is_envoy_sending_413:
 
-Why is Envoy sending 413s?
-==========================
+为什么 Envoy 会发送 413s ？
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
为什么 Envoy 会发送 413 ？
```

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+================================
 
-Envoy by default imposes limits to how much it will buffer for a given request. Generally, Envoy filters are designed to be streaming, and will pass data from downstream to upstream, or will simply pause processing while waiting for an external event (e.g. doing auth checks). Some filters, for example the buffer filter, require buffering the full request or response. If a request body is too large to buffer, but buffering is required by the filter, Envoy will send a 413. The buffer limits can be increased at the risk of making OOMs more possible. Please see the ref:`flow control docs <faq_flow_control>` for details.
+默认情况下，Envoy 会限制它对给定请求的缓冲。
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
默认情况下，Envoy 会对给定的请求限制使用缓存大小。
```

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+一般来说，Envoy 过滤器被设计为流式，将数据从上游传递到下游，或者在等待外部事件时暂停处理（例如，执行认证检查）。
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
一般来说，Envoy 过滤器被设计为流式，将数据从下游传递到上游，或者在等待外部事件时暂停处理（例如，执行认证检查）。
```

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
+对于一些过滤器，例如缓冲过滤器，需要缓冲完整的请求或响应。
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
一些过滤器，例如缓存过滤器，需要缓存完整的请求或响应。
```

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 5]
```diff
@@ -1,6 +1,11 @@
...
+如果一个请求体太大而无法缓冲，但过滤器仍然需要缓冲，那么 Envoy 就会发送 413。
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
如果一个请求体太大而无法缓存，但过滤器仍然需要缓存，那么 Envoy 就会发送 413。
```

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst [hunk 6]
```diff
@@ -1,6 +1,11 @@
...
+缓冲区限制可以增加，但这也会增加 OOMs 的可能性。
```
Comment (helight) [inline docs/root/faq/debugging/why_is_envoy_sending_413s.rst]:
```suggestion
缓存区限制可以加大，但这也会增加 OOM 的可能性。
```

**Review by helight — COMMENTED** [review]
一点建议，看看哈

File: docs/root/faq/debugging/why_is_envoy_sending_413s.rst (hunk 6 above)
Comment (vgbhfive) [reply docs/root/faq/debugging/why_is_envoy_sending_413s.rst]: 感谢指正。

**Review by helight — APPROVED** [review]
LGTM

**Review by addozhang — APPROVED** [review]
/LGTM

**Pull request merged**